test/org/sagebionetworks/bridge/models/CriteriaUtilsTest.java

```diff
@@ -0,0 +1,92 @@
+package org.sagebionetworks.bridge.models;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Set;
+
+import org.junit.Test;
+
+import org.sagebionetworks.bridge.models.schedules.ScheduleContext;
+
+import com.google.common.collect.Sets;
+
+public class CriteriaUtilsTest {
```

Comment:
Is it worth adding a unit tests that has a context with no app version or data groups, just to make sure CriteriaUtils doesn't crash?